client/my-sites/google-my-business/select-business-type/index.js

```diff
@@ -82,22 +96,40 @@ class GoogleMyBusinessSelectBusinessType extends Component {
 	};
 
 	renderLocalBusinessCard() {
-		const { canUserManageOptions, translate } = this.props;
+		const { canUserManageOptions, googleMyBusinessLocations, siteSlug, translate } = this.props;
+		const { locationsChanged } = this.state;
 
 		let connectButton;
 
 		if ( config.isEnabled( 'google-my-business' ) && canUserManageOptions ) {
-			connectButton = (
-				<KeyringConnectButton
-					serviceId="google_my_business"
-					onClick={ this.trackCreateYourListingClick }
-					onConnect={ this.handleConnect }
-				>
-					{ translate( 'Create Your Listing', {
-						comment: 'Call to Action to add a business listing to Google My Business',
-					} ) }
-				</KeyringConnectButton>
-			);
+			if ( googleMyBusinessLocations.length > 0 && ! locationsChanged ) {
+				connectButton = (
+					<Button
+						primary
+						href={ `/google-my-business/select-location/${ siteSlug }` }
+						onClick={ this.trackCreateYourListingClick }
+					>
+						{ translate( 'Select Google My Business Location', {
+							comment:
+								'Call to Action to choose from existing connected Google My Business business listings',
+						} ) }
+					</Button>
+				);
+			} else {
+				connectButton = (
+					<KeyringConnectButton
+						serviceId="google_my_business"
+						onClick={ this.trackCreateYourListingClick }
+						onConnect={ this.handleConnect }
+						primary
+					>
+						{ translate( 'Connect to Google My Business', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Connect to your Google My Business account' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/gmb-flows)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).